### DISCUSSION TITLE
I need a callback function to know when an object moved by MovedNetworkObjects is done moving.

### DISCUSSION THREAD

**Discussion by seto22711** [FirstGearGames/FishNet#519] [Feature Request]
I would like to give an initialization instruction to the object at the time the scene switch is completed, any other ideas?
If there is no such functionality, I would like to have the functionality described in the text of the title.

**Comment by FirstGearGames** [comment]
The move occurs prior to the OnActiveSceneSet callback, which happens after all scenes are loaded but before on load completion callbacks.
